.circleci/config.yml

```diff
@@ -46,9 +53,12 @@ workflows:
   test-deploy:
     jobs:
       - build-browser-sdk
-#      - deploy-browser-sdk:
-#          requires:
-#           - build-browser-sdk
-#         filters:
-#           branches:
-#             only: "master"
\ No newline at end of file
+      - hold:
+          type: approval
+          requires:
+            - build-browser-sdk
+          <<: *masterBranchOnly
+      - deploy-browser-sdk:
+          requires:
+            - hold
+          <<: *masterBranchOnly
```

Comment:
nit: requiring hold which only runs on master should make this redundant

Reply:
good call. Fixed